CHANGES

```diff
@@ -1,3 +1,7 @@
+trunk (unreleased)
+* Update to use raw files only
+* Replace OCaml code with python
```

Comment:
Let's discuss this over :beers: sometime. Have you seen the xenvm repo? That's where the action is now :)